docs/user-guide/complementary-tools.md

```diff
@@ -1,15 +1,17 @@
 # Complementary tools
 
-The linter works well with:
-
 ## Editor plugins
 
 * [linter-stylelint](https://github.com/AtomLinter/linter-stylelint) - An Atom Linter plugin for stylelint.
 * [SublimeLinter-contrib-stylelint](https://github.com/kungfusheep/SublimeLinter-contrib-stylelint) - A Sublime Text plugin for stylelint.
 * [vscode-stylelint](https://github.com/shinnn/vscode-stylelint) - A Visual Studio Code extension for stylelint.
 
+## Runners/loaders
+
+* [gulp-stylelint](https://github.com/olegskl/gulp-stylelint) - A gulp plugin for stylelint.
+* [stylelint-loader](https://github.com/adrianhall/stylelint-loader) - A webpack loader for stylelint.
+
 ## Other linters & formatters
 
 * [cssfmt](https://github.com/morishitter/cssfmt) - A tool that automatically formats CSS and SCSS source code.
-* [postcss-bem-linter](https://github.com/postcss/postcss-bem-linter) - A BEM linter for CSS.
```

Comment:
I would rather keep it in here, because it really is an independent tool, and people might be interested in using it separately.